optimade/filtertransformers/mongo.py

```diff
@@ -38,12 +38,15 @@ def postprocess(self, query):
             query = self._apply_length_aliases(query)
             query = self._apply_aliases(query)
 
+        query = self._apply_relationship_filtering(query)
         query = self._apply_length_operators(query)
 
         return query
 
     def transform(self, tree):
-        return self.postprocess(super().transform(tree))
+        res = self.postprocess(super().transform(tree))
+        print(res)
+        return res
```

Comment:
Either revert or log at debug level. I have made an issue for this as well (#242). We could name the logger `"optimade"` or something?

Reply:
Whoops, obviously left this is in accidentally! I'll leave logging for another PR, don't think this necessarily needs to be added to a debug print.